commit 32515c12d9c53aaccbdcbcb0359a129e6c324814 (refs/changes/77/2877/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2017-10-26T17:10:30-04:00 (1 year, 11 months ago)
    
    joyent/node-vmadm#4 Release version 1.0.0
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index fec47ec..7207cb0 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "vmadm",
-  "version": "0.2.0",
+  "version": "1.0.0",
   "description": "Node.js wrapper around SmartOS vmadm(1M)",
   "main": "lib/index.js",
   "scripts": {
